src/main/java/redis/clients/jedis/params/SetParams.java

```diff
@@ -69,6 +79,9 @@ public SetParams xx() {
     if (contains(XX)) {
       byteParams.add(SafeEncoder.encode(XX));
     }
+    if (contains(GET)) {
+      byteParams.add(SafeEncoder.encode(GET));
+    }
```

Comment:
Move this `if` block after all `if` blocks.

Reply:
done.
btw, does the order of the if block here need to follow the order supported by the version?

Reply:
not necessarily. just being safe here